src/renate/benchmark/experiment_config.py

```diff
@@ -317,6 +318,8 @@ def _get_normalize_transform(dataset_name):
 
 def train_transform(dataset_name: str) -> Optional[Callable]:
     """Returns a transform function to be used in the training."""
+    if dataset_name == "fmow":
+        return FMoW.default_transform
```

Comment:
There seems to exist a `default_transform(datasetname)` in the package. Any reason to go this route?

Reply:
good point